lib/seattleflu/id3c/cli/command/etl/redcap_det_scan.py

```diff
@@ -545,14 +595,22 @@ def create_specimen(record: dict, patient_reference: dict) -> tuple:
     )
 
     # YYYY-MM-DD in REDCap
-    collected_time = record['collection_date'] or None
+    # The `collection_date` field is being removed from the SCAN REDCap projects
+    # on 22 July 2020.
+    #   -Jover, 16 July 2020.
+    collected_time = record.get('collection_date')
 
     # YYYY-MM-DD HH:MM:SS in REDCap
-    received_time = record['samp_process_date'].split()[0] if record['samp_process_date'] else None
+    # `samp_process_date`field does not exist in new SCAN In-Person Enrollments
+    #   -Jover, 17 July 2020
+    received_time = record['samp_process_date'].split()[0] if record.get('samp_process_date') else None
 
     note = None
 
-    if record['able_to_test'] == 'no':
+    # The new SCAN In-Person Enrollment project does not have a
+    # post_collection_data_entry_qc instrument, so it won't have the `able_to_test`field.
+    #   -Jover, 16 July 2020
+    if record.get('able_to_test') == 'no':
```

Comment:
I think for compliance we actually _need_ to have "able to test?" recorded digitally and available. I've asked Peter about how they want to handle it going forward, but my hope is that we'll get an `able_to_test` in this new project too.

Reply:
We're going to get an `able_to_test` field in the new project after all: https://seattle-flu-study.slack.com/archives/C0173B4S1LY/p1595444176252300?thread_ts=1595443935.249500&cid=C0173B4S1LY